docs/wiki/research/roadmap.md

```diff
@@ -17,43 +17,44 @@ The discussion, resources and all research and development on the core protocol
-| Quantum-safe signatures              |                               Proactive research and integration of quantum-resistant cryptographic algorithms                               |                                                                                               Quantum-safe, aggregation-friendly signatures will enhance protocol security against quantum attacks                                                                                                | in research </br> (i) lattice-based[^10] </br>(ii) STARK-based [^11] systems                                      |
+| Upgrade                              | Track |          Topic           |                                                                 Description                                                                  |                                                                                                                                          Expected effect                                                                                                                                          | State of the art                                                                                                  |
+| :----------------------------------- | :---: | :----------------------: | :------------------------------------------------------------------------------------------------------------------------------------------: | :-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :---------------------------------------------------------------------------------------------------------------- |
+|                                      |       |                          |                                                                                                                                              |                                                                                                                                                                                                                                                                                                   |
+| Single slot finality (SSF)           |   -   | Increase validator count |                                           Blocks could be proposed and finalized in the same slot                                            | (i) more convenient for apps (transactions finalization time improved by an order of magnitude, i.e. 12 seconds instead of 12 minutes means better UX for all), (ii) much more difficult to attack (multi block MEV re-orgs can be eliminated and the complexity in consensus mechanism, reduced) | in research </br>(i) VB's SSF notes[^4] </br>(ii) 8192 signatures post-SSF[^5] </br>(iii) simple SSF protocol[^6] |
```

Comment:
One important part here could be a faster bridging for L2s

Reply:
agree, and I will add it as an expected effect of ssf, but I would have to add the context of universal synchronous composability across rollups, because until the challenge period can be drastically reduced for rollups (e.g by real-time SNARK proving,  L2>L1 compatibility via L1 sequencers), SSF won't do much in bridging time, imo. Most likely tho, real-time proving will be achieved before ssf..